scitt/decode_event.py

```diff
@@ -0,0 +1,91 @@
+""" Module for decoding the event """
+
+import argparse
+
+import json
+import base64
+
+from pprint import pprint
```

Comment:
I have found that using json to print structs looks better than pprint. As long as all the dicts are representable in JSON. YMMV. So in pysdk:

     LOGGER.info("Response %s", json.dumps(e, indent=4))

Ignore this if you want

Reply:
going to leave as pprint at the moment, slightly preferred pprint over json dumps :) but if there is a need happy to swap them over!

Good comment though thanks!